### PR TITLE
Scala 2.13.3

### PR DESCRIPTION
**Pull request by avdv**
Scala 2.13.3 has been released, the semanticdb scalac plugin for it is already available.

### CODE REVIEW

**Comment by avdv** [comment]
Rebased onto master to resolve conflicts.

File: build.sbt [hunk 1]
```diff
@@ -22,8 +22,8 @@ developers := List(
   )
 )
 
-scalaVersion := "2.13.2"
-crossScalaVersions := Seq("2.11.12", "2.12.10", "2.12.11", "2.13.1", "2.13.2")
+scalaVersion := "2.13.3"
+crossScalaVersions := Seq("2.11.12", "2.12.10", "2.12.11", "2.13.1", "2.13.2", "2.13.3")
```
Comment (mccartney) [inline build.sbt:26]: Should we drop `2.13.1` then?

Comment (avdv) [reply build.sbt:26]: not sure about that... is it possible anyone is stuck on 2.13.1 ? Probably not.

Comment (mwz) [reply build.sbt:26]: Yeah, we agreed to publish only for 2 latest versions of 2.12 and 2.13, so I'd also be in favour of removing 2.13.1 before the next release.

Comment (mccartney) [reply build.sbt:26]: https://github.com/sksamuel/scapegoat/pull/401

**Pull request merged**